### ISSUE TITLE
Issue with category options

### ISSUE THREAD

**Issue by chloedenavit** [abyot/sun-inventory#126]
Hi Abyot,

I have removed "World blank group" and reordered the list, as you can see here:
![image](https://cloud.githubusercontent.com/assets/21313941/22063626/0c8ec374-dd80-11e6-8349-26cac7c15f21.png)

I cleared the cache several times, but the "world bank group" is still showing up, and the order is not followed. See here:

![image](https://cloud.githubusercontent.com/assets/21313941/22063556/b7658a72-dd7f-11e6-9d8d-363ea743d279.png)

The data element has the correct Category combination:
![image](https://cloud.githubusercontent.com/assets/21313941/22063611/f4d8c6e4-dd7f-11e6-8986-cedfdc7f6fdd.png)

I also tried to delete the World Bank Group category option entirely but it says I cannot.

**Comment by chloedenavit** [comment]
I am having a similar issue with question 5.3.2. UN Agencies involved:
![image](https://cloud.githubusercontent.com/assets/21313941/22064502/2bf3a5dc-dd84-11e6-8a72-f65a829555c4.png)


But:
![image](https://cloud.githubusercontent.com/assets/21313941/22064516/36334282-dd84-11e6-8959-e27dba805936.png)

**Comment by chloedenavit** [comment]
Hi @abyot, just checking in to see if you know what is going on here. Thanks for your help!

**Comment by abyot** [comment]
The two are connected ... order is not respected and agency still remains in the list.

Behind the scene - we have two lists.

The first list is the one you can control through the option list of a category (those listed under "UN Agencies involved" or "UN Agencies (Chair/Focal Points)"). The other one is list of category option combination - which is the list used in data storage / processing. It is not possible to control this list directly. The system picks options of those category(ies) for the category combination and generates corresponding list of category option combination.

Every time you add / remove a category option from a category, the system tries to update the category option combination list automatically. If it fails to do for some reason, you can trigger the update process by going to  

Apps -> Data Administration -> Maintenance -> Update category option combinations

However, the category option combination list could fail to update or stay in synch - if there is a value stored for a given category option combination but category options used to generate the category option combination are removed from a category. This has happened in your particular case - both ILO and World Bank Group were removed from the category but there was data registered for them.....

In short, normally adding option to a category is not problem. A problem is when removing option from a category.... again the problem is if there happens to be data registered for the option to be removed. We don't have a system in place that removes data automatically when an option is removed. One needs to clear data manually before removing an option.

For 5.3.2, you also need to maintain / update a category / combo called "Joint Programme Dimensions". 

I have made some cleanups ... it seems ok now.
